Checking for exitence of the define is not enough since
it will be defined with 0 or 1 value

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>
--- a/sys-utils/ldattach.c
+++ b/sys-utils/ldattach.c
@@ -242,7 +242,7 @@ static int my_cfsetspeed(struct termios
 	 * -- we have to bypass glibc and set the speed manually (because glibc
 	 *    checks for speed and supports Bxxx bit rates only)...
 	 */
-#ifdef _HAVE_STRUCT_TERMIOS_C_ISPEED
+#if _HAVE_STRUCT_TERMIOS_C_ISPEED
 # define BOTHER 0010000		/* non standard rate */
 	dbg("using non-standard speeds");
 	ts->c_ospeed = ts->c_ispeed = speed;
